prac_06/guitars.py

```diff
@@ -0,0 +1,33 @@
+from prac_06.guitar import Guitar
+
+
+def main():
+    guitars = []
+
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $ "))
+        guitars.append(Guitar(name, year, cost))
+        print(Guitar(name, year, cost), "added.")
+        name = input("Name: ")
+
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+
+    if guitars:
+        i = 0
+        print("These are my guitars: ")
+        for guitar in guitars:
+            vintage_string = ""
+            if guitar.is_vintage():
+                vintage_string = "(vintage)"
+                print("Guitar {}: {} ({}), worth ${:5,.2f}{}".format(i + 1, guitar.name, guitar.year, guitar.cost,
+                                                                     vintage_string))
+                i = i + 1
```

Comment:
So you might want to move/change this. 
The reason why, (based of your code right now). if you input the details of a guitar and it isn't vintage. it will print like so:
 These are my guitars: 
Guitar 1: Straton (2011), worth $1,234.00
Guitar 1: Gibson L-5 CES (1922), worth $16,035.40(vintage)
Guitar 2: Line 6 JTV-59 (2010), worth $1,512.90

So whenever a guitar isn't vintage, the code where you increase i will not run and it will have the same number for the next output like above. Suggestion: You could move it out of the "IF" statement and place it after the "ELSE" statement alone or before the "IF" so it always adds on, regardless if vintage or not.

Also you can write "i += 1" and it does the same thing as i = i + 1
Another way is to follow the end of the practical question where it shows how to use it in the for loop:
for i, guitar in enumerate(guitars):